src/m2n/M2N.hpp

```diff
@@ -182,6 +191,15 @@ class M2N
   bool _isMasterConnected = false;
 
   bool _areSlavesConnected = false;
+
+
+  /// between two serial participants, only use the master-master com (required only in some unit tests)
+  bool _useOnlyMasterCom = false;
+
+  // to set _useOnlyMasterCom to true for unit tests
+  friend struct CplSchemeTests::SerialImplicitCouplingSchemeTests::testConfiguredAbsConvergenceMeasureSynchronized;
+  friend struct CplSchemeTests::CompositionalCouplingSchemeFixture;
+  friend struct CplSchemeTests::ExplicitCouplingSchemeFixture;
```

Comment:
This invites to explode. A `friend` testing fixture would be better to handle this in the unit tests.

You can have a look at the use of the `WhiteboxAccessor`:
https://github.com/precice/precice/blob/a5bc1e7652ed957d817c9fcb46848fd760a474fc/src/testing/Testing.hpp#L196

Reply:
Done.